### DISCUSSION TITLE
namespace "spdlog" has no member "stdout_color_mt"	error

### DISCUSSION THREAD

**Discussion by petermeshack** [gabime/spdlog#2882] [General]
what should i do when i get the error namespace "spdlog" has no member "stdout_color_mt"	 example in
#include "Log.h"

namespace Hedric {
	std::shared_ptr<spdlog::logger> Log::s_CoreLogger;
	std::shared_ptr<spdlog::logger> Log::s_ClientLogger;

	void Log::Init() {
		spdlog::set_pattern("%^[%T] %n: %v%$");
		s_CoreLogger = spdlog::stdout_color_mt("Hedric");
		s_ClientLogger = spdlog::stdout_color_mt("App");

	}
}

**Comment by tt4g** [comment]
Missing `#include "spdlog/sinks/stdout_color_sinks.h"`.
